docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,288 @@
+
+The following `devDependencies` are required:
+
+```json
+"devDependencies": {
+    "@dhis2/cypress-commands": "^#.#.#",
+    "@dhis2/cypress-plugins": "^#.#.#",
+},
+```
+
+And it is recommended to add some convenience commands to the `scripts` section, for example:
+
+```json
+"scripts": {
+    "cy:start": "BROWSER=none yarn start",
+    "cy:open": "wait-on 'http-get://localhost:3000' && cypress open",
+    "cy:run": "wait-on 'http-get://localhost:3000' && cypress run",
+    "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+    "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
+},
+```
+
+_Note that `cy:stub` here is meant to perform a local stub-run, the stub run on CI should be configured differently as explained in the [GitHub workflow](#github-workflow) section._
+
+Some notes:
+
+1. These example scripts assume both `wait-on` and `concurrently` are installed available. If this is not the case, run:
+    ```bash
+    yarn add -D wait-on concurrently
+    ```
+1. As an alternative to using `wait-on` and `concurrently` the package `start-server-and-test` could be used, which is recommended by cypress. An example can be found [here](https://github.com/dhis2/scheduler-app/blob/master/package.json#L14-L16).
+1. Setting `BROWSER=none` before running `yarn start` simply means that no browser tab is going to be opened/activated once the app is ready.
+1. In the example above no `dhis2ApiVersion` is passed as a cypress env variable, even though the Network Shim relies on this. This works because that variable is declared in `./cypress.env.json` so that value is used. See the [cypress docs](https://docs.cypress.io/guides/guides/environment-variables) for more info.
+1. The example above only takes into account testing against a single backend version. If an additional v36 version of DHIS2 core was available on `http://localhost:8081` and the test-suite should be executed against that backend as well, the following scripts could be added:
+    ```json
+    "cy:capture-v36": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' --kill-others --success first",
+    "cy:stub-v36": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' --kill-others --success first",
+    ```
+
+### GitHub workflow
+
+At DHIS2 we use GitHub Actions for CI/CD, and we provide a workflow template for DHIS2 platform apps [here](https://github.com/dhis2/workflows/blob/master/ci/dhis2-verify-app.yml#L73-L105). A typical DHIS2 platform-app will probably use the full `dhis2-verify-app` workflow and store this in `./.github/workflows/dhis2-verify-app.yml`. Once the Network Shim has been configured and a capture run has been exectued to create the network fixtures all that is needed is to uncomment the e2e job.
```

Comment:
I'm thinking maybe we should link to a specific commit here as well. To ensure it's stable. And maybe then a link to the repo, stating: `Our workflows are subject to change. For the latest version of our workflows see: ...`

Reply:
exectued -> executed

Reply:
Was little hard to process because it's quite a long sentence, condensed it a little:

> Once the Network Shim has been configured and a capture run has been executed, all that is needed is to uncomment the e2e job.